docs/manifest.json

```diff
@@ -571,6 +571,14 @@
             {
               "title": "Build indicator",
               "path": "/docs/api-reference/next.config.js/build-indicator.md"
+            },
+            {
+              "title": "Turbopack Loaders",
+              "path": "/docs/api-reference/next.config.js/turbopack-loaders.md"
+            },
+            {
+              "title": "Resolve Alias",
+              "path": "/docs/api-reference/next.config.js/resolve-alias.md"
```

Comment:
Should this be called `turbopack-resolve-alias` to make it clear its turbopack only?